GreetingTest.java

```diff
@@ -16,13 +16,13 @@ public void setUp() {
 
     @After
     public void tearDown() {
-        System.setOut(null);
+        PrintStream originalOut = System.out;
```

Comment:
No, I changed the System.out to a PrintStream/ByteArrayStream in setUp; but I see that I should probably save the originalOut during the setUp before the test. Is the changed file what you were looking for? Thank you!